DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/SectionContentViewModelTests.cs

```diff
@@ -509,10 +525,17 @@ public void Diagnostic_assessment_completion_status_does_not_use_plural_if_attem
                 secOutOf: secOutOf);
 
             // When
-            var sectionContentViewModel = new SectionContentViewModel(sectionContent, CustomisationId, SectionId);
+            var sectionContentViewModel = new SectionContentViewModel(config, sectionContent, CustomisationId, SectionId);
 
             // Then
             sectionContentViewModel.DiagnosticCompletionStatus.Should().Be($"{secScore}/{secOutOf} - {diagAttempts} attempt");
         }
+
+        // Test for diagnostic separator if only diagnostic assessment and consolidation present
+        // Test for tutorial separator if tutorials and consolidation present but no post learning assessment
+        // Test for post learning separator if post learning and consolidation present
+        // Test for no diagnostic separator if tutorial, post learning and consolidation are not shown
+        // Test for no tutorial separator if post learning and consolidation are not shown
+        // Test for no post learning separator if consolidation is not shown
```

Comment:
I'd prefer for this not to be committed and pushed. Feel free to keep it locally if it's useful 🙂